compiler/control/OMROptions.cpp

```diff
@@ -3403,22 +3404,22 @@ OMR::Options::processOptionSet(
             endLine=value;
 
             //assume this is the ending ']';
-            endOpt = options+1;
+            endOpt = (char *)options + 1;
             }
          else if (!strnicmp_ignore_locale(options, EXCLUDED_METHOD_OPTIONS_PREFIX, strlen(EXCLUDED_METHOD_OPTIONS_PREFIX)))
             {
             // Option subset that applies to excluded methods
             //
-            filterHeader = options;
-            endOpt = options + strlen(EXCLUDED_METHOD_OPTIONS_PREFIX);
+            filterHeader = (char *)options;
+            endOpt = (char *)options + strlen(EXCLUDED_METHOD_OPTIONS_PREFIX);
             }
          //filter starts with a limitfile subset index
          else if (*options >= '0' && *options <= '9')
             {
-            filterHeader = options;
+            filterHeader = (char *)options;
             // Option subset represented by an index (used in limitfiles)
             //
-            endOpt = options+1;
+            endOpt = (char *)options + 1;
```

Comment:
Several more casts that shouldn't be needed, as `filterHeader` and `endOpt` are `const char *`.